ansible/roles/operator-pipeline/templates/openshift/tasks/yaml-lint.yml

```diff
@@ -22,7 +22,7 @@ spec:
       default: ["--help"]
   steps:
     - name: lint-yaml-files
-      image: docker.io/cytopia/yamllint:1.23@sha256:e9deea51bbdc35950cacf6594dab94100d9bbf5c987540b8874167d58e6bdce2 #tag: 1.23
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
```

Comment:
Same here.

Reply:
It was decided that we'll pin this image by digest closer to the initial release to aid with rapid development.